components/Layout/FormStyles/StyledInput/index.js

```diff
@@ -0,0 +1,14 @@
+import color from "../../Colors";
+import styled from "styled-components";
+
+const StyledInput = styled.input`
+  width: 100%;
+  padding: 8px;
+  margin: 0;
+  border: 1px solid ${color.grey[300]};
+  border-radius: 4px;
+  box-sizing: border-box;
```

Comment:
I guess you already have reset the box sizing in your globalStyles. If not I highly recommend this!